scripts/riscv_setting.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+SETTING_SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+RISCV_LINUX_WORK_DIR=$SETTING_SCRIPT_DIR/..
+
+PREBUILT_DIR=$RISCV_LINUX_WORK_DIR/Prebuilt
+BUILD_DIR=$RISCV_LINUX_WORK_DIR/build/
+
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
```

Comment:
Quote the variable, prefer style `if [[ "${my_var}" == "some_string" ]]`.

https://google.github.io/styleguide/shellguide.html#quoting

Reply:
Done.